src/views/Public/Pages/Records/View.vue

```diff
@@ -20,8 +19,12 @@
       @reload="loadRecord()"
     />
 
-    <portal to="toolbar">
+    <portal
+      class="w-100"
+      :to="recordToolbar"
+    >
       <record-toolbar
+        :show-record-modal="showRecordModal"
```

Comment:
We try to keep the order of props the same as they are defined in component for easier debugging.